opm/output/eclipse/Summary.cpp

```diff
@@ -265,6 +269,23 @@ inline quantity duration( const fn_args& args ) {
     return { args.duration, measure::time };
 }
 
+quantity rpr(const fn_args& args) {
+    const auto& pair = args.regionCells.find( args.num );
+    if (pair != args.regionCells.end()) {
```

Comment:
General tip/preference: negate the check and exit early to save indentation and to avoid having to maintain context and state while reading the function.

```
if( exit_condition ) return;

/*
    don't have to wonder if state update here is needed after the block.
*/
```

Instead, now I have to read as this.

```
if( condition ) {
   /* some state is modified */
} else {
   /* some other state modification */
}

/* is code here dependent on which code path was chosen earlier? */
```
By using the former, the latter question just doesn't surface. Also, it's in essence a short circuit and I strongly prefer when totally disjoint code path nodes don't share a child node, simply because I need to trace both paths to figure out what may or may not have happened.

Two cents etc. and not that big of a deal for such a small function.

YMMV.